app/controllers/settings.js

```diff
@@ -32,6 +32,15 @@ export default Controller.extend({
     return isPrivate && settings.hasOwnProperty('allow_config_imports');
   }),
 
+  showBetaFeatures: computed('showConfigValidationSwitches', function () {
+    return this.get('showConfigValidationSwitches');
+  }),
+
+  showConfigValidationSwitches: computed('model.settings', function () {
+    let settings = this.get('model.settings');
+    return settings.hasOwnProperty('config_validation');
+  }),
```

Comment:
Similarly, some of the computed property functions that ember provides could simplify this, although, it might require 2 of them. Something like:
```
  hasNoConfigValidation: none('model.settings.config_validation'),
  showConfigValidationSwitches: not('hasNoConfigValidation'),
```

We've been trying to move more towards using these when possible. Here's the docs for these functions https://api.emberjs.com/ember/3.12/modules/@ember%2Fobject#functions-computed

Reply:
done 🎉